CodeEdit/NavigatorSidebar/FindNavigator/FindNavigatorResultFileItem.swift

```diff
@@ -32,17 +35,15 @@ struct FindNavigatorResultFileItem: View {
              self.jumpToFile = jumpToFile
     }
 
+    @ViewBuilder
     private func foundLineResult(_ lineContent: String?, keywordRange: Range<String.Index>?) -> some View {
-        guard let lineContent = lineContent, let keywordRange = keywordRange else {
-            return AnyView(EmptyView())
-        }
-        return AnyView(
+        if let lineContent = lineContent, let keywordRange = keywordRange {
```

Comment:
Please add a line break for multiple if let statements:

```swift
if let lineContent = lineContent, 
   let keywordRange = keywordRange {
    ...
}
```

Reply:
Okay, got it

Reply:
Added to pr